src/pretender.es.js

```diff
@@ -0,0 +1,950 @@
+import self$1 from './iife-self-placeholder.js';
```

Comment:
Is this a build output? Should it be checked into the repo?

Reply:
Nope, build output removed.